enterprise/admin/forms.py

```diff
@@ -90,6 +90,12 @@ class ManageLearnersForm(forms.Form):
         ],
     )
     reason = forms.CharField(label=_("Reason for manual enrollment"), required=True)
+    discount = forms.IntegerField(
```

Comment:
On the other work we did, there is the requirement for the ability to store up to 5 decimal places. Because of this, I believe you will need to use a DecimalField here